.github/workflows/build-and-push-containers.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and push containers
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release-container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Compile tag list
```

Comment:
you could use https://github.com/crazy-max/ghaction-docker-meta here

Reply:
True - but this is fast and doesn't require any additional overhead as it's using tooling already in the environment.

Thanks for the link, though - I'd not seen that one!